celements-user-manager/web-module/src/main/webapp/resources/celRes/userAdmin/dropdownMenu.css

```diff
@@ -0,0 +1,46 @@
+div.userAdminContent div.searchUser div > div.dropdown  {
+    display: block;
+    width: 60px;
+    float: right;
+    position: relative;
+    margin: 15px 0px 0px;
+}
+
+div.userAdminContent div.searchUser div > div.dropdown .dropbtn {
+    display: block;
+    border: 2px solid #156eb9;;
+    color: #156eb9;;
+    font-size: 14px;
+    text-align: center;
+    font-weight: bolder;
+    padding: 0px 0px 6px;
+    margin: 1px 0px;
+}
+
+div.userAdminContent div.searchUser div > div.dropdown:hover div.dropdown-content {
+    display: block;
+}
+
+div.userAdminContent div.searchUser div > div.dropdown div.dropdown-content {
+    display: none;
+    background-color: whitesmoke;
+    min-width: 160px;
+    position: absolute;
+    right: 0;
+    border-left: 0.5px solid black;
+    border-right: 0.5px solid black;
+    border-bottom: 0.5px solid black;
+    z-index: 1000;
+}
+
+div.userAdminContent div.searchUser div > div.dropdown div.dropdown-content a {
```

Comment:
consider using an id as the anchor e.g.
`#userAdminActionMenu`
instead of
`div.userAdminContent div.searchUser div > div.dropdown`